From 7ae60785d80c7dc9750918c53929d3a7f403027d Mon Sep 17 00:00:00 2001
Message-Id: <7ae60785d80c7dc9750918c53929d3a7f403027d.1423872525.git.feitong.yi@intel.com>
In-Reply-To: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
References: <3062c7e6b919c393846ff0e13558e7f648b352ca.1423872525.git.feitong.yi@intel.com>
From: Deepak S <deepak.s@intel.com>
Date: Sat, 31 Jan 2015 13:48:42 +0530
Subject: [PATCH 91/95] Revert "REVERTME [VPG]: drm/i915/vlv: Perform modeset
 based on DPMS state during resume"

Reverting below commit and follwoing patch will have generic implementation for all platform.

The Orginal issue was introduced in:
  commit 0e070addb68cf4b8d0ff6b34ee75ba8880510949
  Author: Akash Goel <akash.goel@intel.com>
  Date:   Wed Oct 29 15:13:50 2014 +0530

  REVERTME [VPG]: drm/i915/vlv: Perform modeset based on DPMS state during resume

Change-Id: I5ea2db82e945cb1c48a1c9923f9720d744c533b9
Signed-off-by: Deepak S <deepak.s@intel.com>
---
 drivers/gpu/drm/i915/i915_drv.c |   25 -------------------------
 1 file changed, 25 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_drv.c b/drivers/gpu/drm/i915/i915_drv.c
index 0f8e6d6..f634256 100644
--- a/drivers/gpu/drm/i915/i915_drv.c
+++ b/drivers/gpu/drm/i915/i915_drv.c
@@ -676,31 +676,6 @@ void intel_console_resume(struct work_struct *work)
 	console_unlock();
 }
 
-static bool display_is_on(struct drm_device *dev)
-{
-	struct drm_connector *connector;
-	bool display_is_on = false;
-
-	drm_modeset_lock_all(dev);
-	list_for_each_entry(connector, &dev->mode_config.connector_list, head) {
-		if (!connector->encoder || !connector->encoder->crtc)
-			continue;
-		/*
-		 * If Display wasn't turned off, before going to suspend then
-		 * it should be re-enabled now, as we don't expect the DPMS ON
-		 * call to come in that case
-		 */
-		if (connector->dpms != DRM_MODE_DPMS_OFF) {
-			DRM_DEBUG_KMS("Display was on before suspend\n");
-			display_is_on = true;
-			break;
-		}
-	}
-	drm_modeset_unlock_all(dev);
-
-	return display_is_on;
-}
-
 static int i915_drm_thaw_early(struct drm_device *dev)
 {
 	struct drm_i915_private *dev_priv = dev->dev_private;
-- 
1.7.9.5

